src/VirtoCommerce.CatalogPersonalizationModule.Web/Localizations/en.VirtoCommerce.CatalogPersonalization.json

```diff
@@ -35,5 +35,21 @@
         "tags": {
           "__any": "Everyone" 
         } 
-    }
+    },
+     "settings": {
+      "CatalogPersonalization": {
+        "ExportImport": {
+          "description": "Module description in platform export/import wizard",
+          "title": "Export/Import description"
+        },
+        "TagsInheritancePolicy": {
+          "description": "DownTree - propagates all tags assigned to categories for all their descendants. UpTree - propagates tags up to hierarchy from descendants to parents",
+          "title": "Selected policy for tags propagation (requires restart)"
+        },
+        "CronExpression": {
+          "description": "x minutes use '0/x * * * *', x hourse use '0 */x * * *',  you can set minutes also '15 */x * * *', x days '0 0 */x * *'   ",
+          "title": "Job cron expression for synchronize the outlines of tagged entities. Valid only when the UpTree propagation policy is selected. (requires restart)"
+        }
+      }
+     }
```

Comment:
Maybe permission localization is missing

Reply:
added